src/local-run.ts

```diff
@@ -1,51 +1,128 @@
-import { createManifest, readAll } from "./deps.ts";
-import { ParsePayload } from "./parse-payload.ts";
-import { DispatchPayload } from "./dispatch-payload.ts";
+import { createManifest, parse } from "./deps.ts";
```

Comment:
the diff for this file is a little confusing, but whats going on is that I'm renaming:

local-run.ts -> local-run-function.ts
local-run.test.ts -> local-run-function.test.ts

and replacing local-run.ts with a wrapper that Deno.runs local-run-function.ts. I mostly did this to keep compatibility with folks that had custom slack.jsons with a different start hook